mne/io/eeglab/eeglab.py

```diff
@@ -129,16 +129,12 @@ def _get_info(eeg, montage, eog=()):
 
     info = create_info(ch_names, sfreq=eeg.srate, ch_types='eeg')
 
-    if not(montage is None and eeg_montage is None):
-        # XXX: This is kept for back compatibility, we should check the logic
-        if eog == 'auto':
-            eog = _find_channels(ch_names)
-
-        for idx, ch in enumerate(info['chs']):
-            ch['cal'] = CAL
-            if ch['ch_name'] in eog or idx in eog:
-                ch['coil_type'] = FIFF.FIFFV_COIL_NONE
-                ch['kind'] = FIFF.FIFFV_EOG_CH
+    eog = _find_channels(ch_names) if eog == 'auto' else eog
```

Comment:
I would rename _find_channels to _find_eog_channels

Reply:
I woudl also make sure that at this stage eog is a list of int (not anymore ch names)

Reply:
I've used the `_find_channels` that we have in utils. I touched nothing. This function is used in other places aswell.

I've modified for `_find_channels(ch_names, ch_types='EOG')` for readability but thats all.

If we want to change its name and behavior we can do it in a separate PR.